test/main.cpp

```diff
@@ -361,10 +359,8 @@ EXPORT int main(int argc, char const* argv[])
 #else
 	process_id = getpid();
 #endif
-	std::string root_dir = current_working_directory();
-	char dir[40];
-	snprintf(dir, sizeof(dir), "test_tmp_%u", process_id);
-	std::string unit_dir_prefix = combine_path(root_dir, dir);
+	std::string root_dir { current_working_directory() };
```

Comment:
if anything, this should be made const. I don't see a strong reason to use uniform initialization here, instead of regular initialization.